config/components/manager/manager.yaml

```diff
@@ -31,7 +31,7 @@ spec:
         - /manager
         args:
         - "--zap-log-level=2"
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
Can we use kustomize patches to change imagePullPolicy?

Reply:
yea I can look into that.

Reply:
I attempted to add kustomization but the solution isn't ideal.  Please feel free to suggest better ideas.  

https://github.com/kubernetes-sigs/kueue/pull/421#discussion_r1001116302